launcher/modplatform/flame/FlameAPI.h

```diff
@@ -1,35 +1,58 @@
 #pragma once
 
 #include "modplatform/helpers/NetworkModAPI.h"
-
+#include "BuildConfig.h"
 class FlameAPI : public NetworkModAPI {
+   private:
+    inline auto getSortFieldInt(QString sortString) const -> int {
+        return sortString == "Featured"
+          ? 1
+          : sortString == "Popularity"
+          ? 2
+          : sortString == "LastUpdated"
+          ? 3
+          : sortString == "Name"
+          ? 4
+          : sortString == "Author"
+          ? 5
+          : sortString == "TotalDownloads"
+          ? 6
+          : sortString == "Category"
+          ? 7
+          : sortString == "GameVersion"
+          ? 8
+          : 1;
+    }
+
    private:
     inline auto getModSearchURL(SearchArgs& args) const -> QString override
     {
         auto gameVersionStr = args.versions.size() != 0 ? QString("gameVersion=%1").arg(args.versions.front().toString()) : QString();
-
+        
         return QString(
-                   "https://addons-ecs.forgesvc.net/api/v2/addon/search?"
+                   BuildConfig.CFPROXY_URL+"/v1/mods/search?"
                    "gameId=432&"
                    "categoryId=0&"
                    "sectionId=6&"
```

Comment:
From what I could gather, [this should be changed to `classId`](https://docs.curseforge.com/#search-mods) now